src/clj/rems/css/styles.clj

```diff
@@ -96,8 +102,8 @@
                            :background-size :contain
                            :background-repeat :no-repeat
                            :background-position [[:center :center]]}]
-                         [:.logo
-                          {:height (u/px 150)}]))
+                         [:.logo {:height logo-height}]
+                         [:.navbar-brand {:display "none"}]))
```

Comment:
Do we really need to set this to `display: none` because aren't we using the `navbar-small` component for the smallest breakpoint?